docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -386,15 +399,17 @@ based on your project's build tool:
 
 
 
-Next, provide a `@BeforeEach` method to configure MockMvc, WebTestClient, or REST Assured:
+Next, you can provide a `@BeforeEach` method to configure MockMvc, WebTestClient, or REST
```

Comment:
I wonder if this should be "should" rather than "can"?  It's something that you have to do, rather than something that you can do if you want to.

Reply:
"must" again.